app/views/candidate_mailer/unconditional_offer_accepted.text.erb

```diff
@@ -2,14 +2,16 @@ Dear <%= @application_form.first_name %>
 
 You’ve accepted <%= @provider_name %>’s offer to study <%= @course_name_and_code %>.
 
-The course starts <%= @start_date %>.
+<%= @provider_name %> will check your references once they receive them.
 
-<%= @provider_name %> will let you know if they need further information before you can start training.
+They’ll let you know if they need further information before you can start training.
 
-#What to do if you’re unable to start training in <%= @start_date %>
+Sign into your account to check the progress of your reference requests.
 
-Some providers allow you to defer your offer. This means that you could start your course a year later.
+[Sign into your account](<%= candidate_magic_link(@application_form.candidate) %>).
+
+#If you’re unable to start training in <%= @start_date %>
```

Comment:
Subheading?
```suggestion
## If you’re unable to start training in <%= @start_date %>
```

Reply:
I used the same syntax as app/views/candidate_mailer/unconditional_offer_accepted.text.erb, which currently renders correctly as a heading.

Reply:
This feels like we should just resolve/not do anything.

Reply:
You're right, I'm wrong. 🤦